tests/providers/amazon/aws/hooks/test_s3.py

```diff
@@ -313,6 +313,29 @@ def test_copy_object_acl(self, s3_bucket):
             assert ((response['Grants'][0]['Permission'] == 'FULL_CONTROL') and
                     (len(response['Grants']) == 1))
 
+    @mock_s3
+    def test_delete_bucket(self, s3_bucket):
+        # assert if the bucket is created
+        conn = boto3.client('s3')
```

Comment:
You don't need `boto3` here. Use `mock_hook.load_file_obj` instead of `conn.upload_fileobj` :)